pin/pin.go

```diff
@@ -221,30 +231,51 @@ func (p *pinner) RemovePinWithMode(key util.Key, mode PinMode) {
 func LoadPinner(d ds.ThreadSafeDatastore, dserv mdag.DAGService) (Pinner, error) {
 	p := new(pinner)
 
+	rootKeyI, err := d.Get(pinDatastoreKey)
```

Comment:
Before this, there were keys like `/local/pins/direct/keys`. This commit changes nothing, in that respect.